lesson07/assignment/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="css/style.css" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
+        <script src="https://unpkg.com/vue@next"></script>
+        <title>Assignment 07 - Using Vue</title>
+    </head>
+    <body>
+        <main class="main" id="app">
+          <h1>Webpage using Bootstrap</h1>
+          <section class="container">
+            <h2>Movie List</h2>
+            <div class="accordion" id="movieListAccordion">
+
+              <!-- start v-for -->
+              <div class="accordion-item" v-for="movie in movieList">
```

Comment:
Great use case for a v-for loop, and good work in making the accordion load dynamically!